### PR TITLE
Add allowed format

### PR DESCRIPTION
**Pull request by shenyangshi**

### CODE REVIEW

**Review by jmosbacher — COMMENTED** [review]
thanks, probably for the default value we can allow all of the formats supported by `get_resource` [here](https://github.com/XENONnT/straxen/blob/master/straxen/common.py)

**Review by jmosbacher — COMMENTED** [review]
also sorry for not noticing this earlier but get_resource actually returns the file contents. to get the file you need:
```python
        downloader = straxen.MongoDownloader()
        return downloader.download_single(name)
```

**Comment by jmosbacher** [comment]
Also you need to import straxen inside the function using it so that we dont make straxen a requirement of xedocs

**Comment by shenyangshi** [comment]
@jmosbacher this branch runs well locally but the test doesn't pass, could you have a look?

**Comment by jmosbacher** [comment]
@shenyangshi see my previous comment, straxen is not a requirement of xedocs so you need to import it inside the function you use it in

File: xedocs/schemas/corrections/base_references.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 from typing import ClassVar, Literal
 from pydantic import constr
 import rframe
+import straxen
```
Comment (jmosbacher) [inline xedocs/schemas/corrections/base_references.py:5]:
```suggestion
```

**Review by jmosbacher — CHANGES_REQUESTED** [review]

**Comment by shenyangshi** [comment]
Sorry I noticed it but somehow my edit was not saved... Now done.

**Review by jmosbacher — APPROVED** [review]

**Comment by shenyangshi** [comment]
Thanks, is it possible to release a new version of xedocs @jmosbacher? So we can bump version in straxen and try the reprocessing tomorrow.

**Comment by jmosbacher** [comment]
sure

**Pull request merged**